python/ray/__init__.py

```diff
@@ -256,12 +255,13 @@ def __getattr__(self, attr):
     # TODO(Clark): Remove this one we drop Python 3.6 support.
     from ray import data  # noqa: F401
     from ray import workflow  # noqa: F401
+    from ray import autoscaler  # noqa: E402,F401
```

Comment:
nit: add a comment why this is here?

Reply:
sure. it is to enable ray.autoscaler.sdk.request_resources I believe